### PR TITLE
Fix build regression for ARM.

### PR DESCRIPTION
**Pull request by benpye**
This fixes a build regression on ARM, and allows mscorlib.dll to be built for ARM64 and ARM on Linux. @mikem8361 's commit 2f44626 appeared to cause the regression.

@mmitche this should fix the error you were seeing.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -218,7 +218,7 @@ for i in "$@"
         ;;
         arm64)
         __BuildArch=arm64
-        __MSBuildBuildArch=arm64
+        __MSBuildBuildArch=AnyCPU
```
Comment (kangaroo) [inline build.sh]: why did you change this to anycpu?

Comment (benpye) [inline build.sh]: In error, sorry, I was looking through and couldn't see any definition for arm64 in MSBuild.

**Comment by jkotas** [comment]
LGTM. Thanks!
